tests/test_meshtastic_utils_connect.py

```
@@ -626,3 +825,1141 @@ def test_none_startup_drain_event_is_safe_noop(reset_meshtastic_globals):
+
+        assert result is None
+        mock_client.close.assert_called()
+        assert mu.meshtastic_client is None
+        assert mu._relay_active_client_id is None
+
+    def test_ble_getMyNodeInfo_failure_triggers_ble_disconnect_cleanup(self):
+        config = _ble_config()
+
+        class _FakeBLEFailsNodeInfo(_FakeBLEInterfaceCompat):
+            def getMyNodeInfo(self) -> dict[str, dict[str, str]]:
+                raise RuntimeError("node info failed")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)